dcap/types/src/quote3.rs

```diff
@@ -215,7 +290,7 @@ impl<'a> TryFrom<&'a [u8]> for SignatureData<'a> {
 
         let (bytes, point_bytes) = take(KEY_SIZE)(bytes);
         let point = EncodedPoint::from_untagged_bytes(point_bytes.into());
-        let attestation_key = VerifyingKey::from_encoded_point(&point).unwrap();
+        let attestation_key = VerifyingKey::from_encoded_point(&point)?;
```

Comment:
This removal of an errant unwrap could be it's own PR, but it's fairly small and was noticed when creating tests for quote verification.